pam_authz/pam/README.md

```diff
@@ -0,0 +1,64 @@
+# Styra's PAM authorization module
+
+This implements a PAM module that integrates with Open Policy Agent (OPA)
+to make authorization decisions. When invoked, the module queries OPA
+and if no errors are returned, then the action is allowed. This allows
+administrators to implement fine-grained access control. For
+example, one can use this module to limit usage of `sudo` to particular
+times of day.
```

Comment:
Might be useful to split this into two parts:

1. Why is pam_authz useful?
2. How does pam_authz work (at a high level)?

I could imagine these under separate headings.